tailwind.js

```diff
@@ -235,6 +235,7 @@ module.exports = {
     'tight': '-0.05em',
     'normal': '0',
     'wide': '0.05em',
+    '1px': '1px'
```

Comment:
I was hesitant to add these values to the different objects as if more and more are added the file size can get too large.  

Curious if I should use the closest value to the ones provided.
letter-spacing: 0.05em is about 0.8px. (original spec had 1px)
borderRadius: .125em is 2px. (original spec had 3px).
heights: 65px ~ 4rem ... 140px = 8.75rem (already has 8 rem in the .js file)